source/Irrlicht/CSkyDomeSceneNode.cpp

```diff
@@ -168,7 +168,9 @@ void CSkyDomeSceneNode::generateMesh()
 		auto vAttr = core::smart_refctd_ptr<video::IGPUBuffer>(SceneManager->getVideoDriver()->createGPUBufferOnDedMem(reqs, true),core::dont_grab);
 		vAttr->updateSubRange(video::IDriverMemoryAllocation::MemoryRange(0, reqs.vulkanReqs.size), vertices);
 		_IRR_ALIGNED_FREE(vertices);
-		vao->setVertexAttrBuffer(core::smart_refctd_ptr<video::IGPUBuffer>(vAttr), asset::EVAI_ATTR0, asset::EF_R32G32B32_SFLOAT, 4 * (3 + 2), 0);
+		vao->
+			

```

Comment:
whats going on

Reply:
hell XD resolved

Reply:
then mark as resolved

Reply:
resolved

Reply:
there's a button for that                                                                                                          |
                                                                                                                                                    |
                                                                                                                                                   V